src/bom/bitcode.rs

```diff
@@ -243,52 +244,61 @@ fn make_bitcode_filename(target : &OsString, bc_dir : &Option<&PathBuf>) -> Path
 /// This handles removing flags that clang can't handle (or that we definitely
 /// do not want for bitcode generation), as well as transforming the output file
 /// path
-fn build_bitcode_arguments(chan : &mut mpsc::Sender<Option<Event>>,
-                           bc_opts : &BCOpts,
-                           orig_args : &[OsString]) -> anyhow::Result<BitcodeArguments> {
+fn build_bitcode_arguments<E>(chan : &mut mpsc::Sender<Option<Event>>,
+                              bc_opts : &BCOpts<E>,
+                              orig_args : &[OsString],
+                              ops : &mut ChainedOps) -> anyhow::Result<OsString>
+where E: OsRun
```

Comment:
Given that there is nothing higher-order going on here, can this just be inlined in the type signature to eliminate the type argument?